docs/support/index.rst

```diff
@@ -9,29 +9,47 @@ Slack
 -----
 
 Most of us use Slack for their online communication with co-workers. We know that it is not convenient to have another chat window open,
-therefore we've chosen Slack for the live communication with the community.
+therefore we've chosen Slack for the live communication with the community. Slack is supposed to handle the most urgent questions
+in the fastest way possible.
 
-We have created following rooms so far:
+The most important rooms for newcomers are:
 
 #general - for discussions about Sylius development itself,
 #docs - for discussions related to the documentation,
 #support - for asking questions and helping others,
 #random - for the non-work banter and water cooler conversation.
 
-If you have any suggestions, please let us know on Slack!
+But there are many more specific channels also.
+If you have any suggestions regarding its organization, please let us know on Slack!
 
 Slack requires inviting new members, but this can be done automatically, just go to `sylius.com/slack <http://sylius.com/slack>`_,
 enter your email and you will get an invitation.
-If possible, please use GitHub username - it will help us to recognize each other easily!
+If possible, please use your GitHub username - it will help us to recognize each other easily!
+
+Forum
+-----
+
+In response to the rapid growth of our Ecosystem, we decided it is necessary to launch a brand new platform for support
+and exchanging experience. On the contrary to Slack, our Forum is much easier to follow, on Slack unless you are a part
+of the discussion when it is happening, it might be difficult to catch up when you have been offline.
+Forum has a search engine so it is convenient to browse for what is interesting for you.
+
+The Sylius Community Forum is meant for everyone who is interested in eCommerce technology and Sylius. We invite both existing
+and new community members to join the discussion and help shape the ecosystem, products & services we are building.
+Get to know other community members, ask for support, suggest an improvement or discuss your challenge.
+
+You can register via email, Twitter & GitHub by going to `https://forum.sylius.com/ <https://forum.sylius.com/>`_ and hitting the “Sign-Up” button.
 
 StackOverflow
 -------------
 
-We encourage asking Sylius related questions on the `stackoverflow.com <http://stackoverflow.com>`_ platform.
+We also encourage asking Sylius related questions on the `stackoverflow.com <http://stackoverflow.com>`_ platform.
 
 * Search for the question before asking, maybe someone has already solved your problem,
 * Be specific about your question, this is what SO is about, concrete questions and solutions,
 * Be sure to tag them with **sylius** tag - it will make it easier to find for people who can answer it.
+* Try also tagging your questions with the **symfony** tag - the Symfony community very big and might be really helpful with
```

Comment:
`the Symfony community very` -> `the Symfony community is very`? Maybe we can just remove `very big and` part?